tools/bridge/bridge/main.py

```diff
@@ -224,6 +224,23 @@ def stop(pool, timeout):
         os._exit(os.EX_SOFTWARE)
 
 
+def reload_logging_config(config_path):
+    gevent.getcurrent().name = "sighup"
+    logger.info(
+        f"Handling sighup, trying to reload the logging configuration from {config_path}"
+    )
+    try:
+        config = load_config(config_path)
+        configure_logging(config)
+        logger.info("Logging has been reconfigured")
+    except Exception as err:
+        # this is a signal handler. make sure we don't die as this
+        # will raise the error in the main greenlet.
+        logger.warning(
```

Comment:
Isn't `warning` too light here? `logger.exception` sounds like the more appropriate handler.

Reply:
I don't want to see a traceback in that case. I think a warning is fine. If you change the logging configuration, you will look at the log output anyway. Maybe I'll change it to logger.critical